### PR TITLE
Add Unicode support

### PR DESCRIPTION
**Pull request by aggFTW**
As reported on http://disq.us/9ntn8x

### CODE REVIEW

**Comment by aggFTW** [comment]
Closes #228

File: remotespark/utils/configuration.py [hunk 1]
```diff
@@ -188,3 +188,7 @@ def default_samplefraction():
 @_override
 def max_slices_pie_graph():
     return 100
+
+
+def pyspark_sql_encoding():
```
Comment (msftristew) [inline remotespark/utils/configuration.py]: Missing `_override` decorator?

**Comment by msftristew** [comment]
Does Scala work?

**Comment by aggFTW** [comment]
Yes

**Comment by msftristew** [comment]
LGTM. We should probably use Unicode for all strings as we continue working on the codebase.

**Comment by guybartal** [comment]
Hi
We have the same problem with Hebrew characters in UTF-8, when are you going to publish the fix? 
it is very important for my customer's POC....

Best Regards,
Guy Bertental

**Comment by msftristew** [comment]
Hi Guy, 

This issue was resolved back on May 9.  The most current version of the code should have this fix (including the currently live version of the code on HDInsight).  Are you running the latest version?

**Comment by guybartal** [comment]
Hi @msftristew 
We have created the cluster today on West Europe region so I guess we do have latest version,
you can see my problem here:
[http://stackoverflow.com/questions/37698276/utf-8-text-in-hdinsight-cluster-with-spark-result-encoding-error-ascii-codec-c](url)

**Comment by msftristew** [comment]
Thanks Guy.  The issue you're describing is distinct from this issue -- this issue prevented people from executing code containing Unicode characters in notebooks.  Your issue is an unrelated string encoding error which doesn't seem to have anything to do with the Sparkmagic project itself.  We'll follow up directly at Stack Overflow.

**Comment by guybartal** [comment]
thanks @msftristew hoping to find solution soon....

**Pull request merged**